msteams-platform/whats-new.md

```diff
@@ -439,6 +439,8 @@ Discover Microsoft Teams platform features that are deprecated. You can now get
 
 Teams platform features that aren't available.
 
+* ***January 25, 2024***: Teams, Outlook, and Microsoft 365 web domains are migrating to the new *cloud.microsoft* domain. The old domains **will be** deprecated on June 1, 2024. Configure your app to run on these new domains before June 1, 2024 to ensure continued functionality across Teams, Outlook, and Microsoft 365. For more information, see [Configure Content Security Policy headers](m365-apps/extend-m365-teams-personal-tab.md#configure-content-security-policy-headers).
```

Comment:
Could we tighten up the wording here, so its short and eye-catching to the reader. Suggestion:

Teams, Outlook, and Microsoft 365 web domains are moving to *cloud.microsoft*. [Update your app before the old domains are deprecated on June 1, 2024](m365-apps/extend-m365-teams-personal-tab.md#configure-content-security-policy-headers) to prevent loss of critical functionality.

Reply:
Done.